org.spoofax.terms/src/org/spoofax/interpreter/terms/ISimpleTerm.java

```diff
@@ -37,26 +37,84 @@
 	 * @param type the term attachment type, or {@code null} if the first attachment should be returned
      * @return the term attachment of the specified type; or {@code null} if it could not be found
 	 */
-    @Nullable <T extends ITermAttachment> T getAttachment(@Nullable TermAttachmentType<T> type);
+    @SuppressWarnings("unchecked")
```

Comment:
This seems to be using tabs? Please use 4 spaces (doesn't the Eclipse formatter do that?)

Reply:
When editing existing file I try to use the indentation that is used in that file rather than reformat everything, to keep the diff smaller. So in this case, that's tabs :(